.changeset/smooth-rings-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/theme': patch
+---
+
+Fixed a bug to support string fontSize values (`"2.5rem"`) instead of forcing numeric-only values & requiring casts. In addition, added an optional fontFamily prop for h1-h6 BackstageTypography variants to allow further customization.
```

Comment:
```suggestion
Added support for string `fontSize` values (e.g. `"2.5rem"`) in themes in addition to numbers. Also added an optional `fontFamily` prop for header typography variants to allow further customization.
```

``` `backticks` ``` on identifiers will silence the spell checker. Also reworded a bit to fit casual adopters reading this later down the line.